code/modules/atmospherics/machinery/components/unary_devices/unary_base.dm

```diff
@@ -9,6 +9,8 @@
 
 	var/datum/pipeline/parent
 
+	var/welded = FALSE
```

Comment:
I'd almost make you refactor the vent and scrubber to have a common parent. But I know the pain that would entail regarding map changes etc. I'm not sure this is the best approach though